.scripts/build.yml

```diff
@@ -1,7 +1,7 @@
 steps:
   - task: NodeTool@0
     inputs:
-      versionSpec: "18.x"
+      versionSpec: "18.19.x"
```

Comment:
Temp downgrade the node version due to the issue https://github.com/Azure/autorest.typescript/issues/2454.